ecs-init/config/common_test.go

```diff
@@ -45,3 +45,20 @@ func TestDockerUnixSocketWithDockerHost(t *testing.T) {
 		t.Error("DockerUnixSocket() should read from envrionment variable DOCKER_HOST, when DOCKER_HOST is set")
 	}
 }
+
+func TestGetS3BucketMapByRegion(t *testing.T) {
+	s3Bucket := getBaseLocationForRegion(DefaultRegionName)
```

Comment:
With go 1.7, you can rewrite this as table driven tests. More info and examples:

* https://blog.golang.org/subtests
* https://github.com/golang/go/wiki/TableDrivenTests
* https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go